Src/AutoFixtureUnitTest/FixtureTest.cs

```diff
@@ -948,8 +948,8 @@ public void InjectCustomUpperLimitWillCauseSutToReturnNumbersInLimit()
             var result = sut.Create<DoublePropertyHolder<int, long>>();
             // Verify outcome
             Assert.True(
-                (result.Property1 >= lower && result.Property1 < upper) &&
-                (result.Property2 >= lower && result.Property2 < upper)
+                (result.Property1 >= lower && result.Property1 <= upper) &&
+                (result.Property2 >= lower && result.Property2 <= upper)

```

Comment:
I agree with @ploeh here. The algorithm's specification must remain the same, otherwise we might end up introducing a breaking change.

What needs to addressed by this Pull Request is that the upper limit's value shouldn't be the same all the time.